### ISSUE TITLE
img missing alt text

### ISSUE THREAD

**Issue by discospiff** [pkkenn/IS7012#1]
Use the alt attribute on the img tag to describe the image to a user who can't see it.  Describe, in a concise way, what the image looks like.

https://github.com/pkkenn/IS7012/blob/e662cc684b5fa8c5cc8cc9a9b4a5562b22f28186/Week 1/Assignment 1 Resume/alexiskh-about-me.html#L10

**Comment by pkkenn** [comment]
[https://github.com/pkkenn/IS7012/blob/7b5374fdd434b0e6d964c188b4cbe99af0a89461/Week%201/Assignment%201%20Resume/alexiskh-about-me.html#L10](https://github.com/pkkenn/IS7012/blob/7b5374fdd434b0e6d964c188b4cbe99af0a89461/Week%201/Assignment%201%20Resume/alexiskh-about-me.html#L10)
